src/Compiler/Microsoft.AspNetCore.Razor.Language/test/DefaultRazorProjectFileSystemTest.cs

```diff
@@ -17,16 +17,25 @@ public class DefaultRazorProjectFileSystemTest
         "TestFiles",
         "DefaultRazorProjectFileSystem");
 
-    [Theory]
-    [InlineData(null)]
-    [InlineData("")]
-    public void NormalizeAndEnsureValidPath_ThrowsIfPathIsNullOrEmpty(string path)
+    [Fact]
+    public void NormalizeAndEnsureValidPath_ThrowsIfPathIsNull()
+    {
+        // Arrange
+        var fileSystem = new TestRazorProjectFileSystem("C:/some/test/path/root");
+
+        // Act and Assert
+        var exception = Assert.Throws<ArgumentNullException>(() => fileSystem.NormalizeAndEnsureValidPath(null!));
+        Assert.Equal("path", exception.ParamName);
```

Comment:
Nit: XUnit has a helper to assert the parameter name for anything that inherits `ArgumentException` like `ArgumentNullException`.
```suggestion
        var exception = Assert.Throws<ArgumentNullException>("path", () => fileSystem.NormalizeAndEnsureValidPath(null!));
```

Reply:
Great suggestion -- thanks! I did a quick Find All References for `ArgumentException.ParamName` to find any other tests that might be able to use this overload.